sdk/communication/communication-chat/test/public/utils/recordedClient.ts

```diff
@@ -71,28 +65,5 @@ export function createChatClient(userToken: string): ChatClient {
   }
   const { url } = parseClientArguments(env.COMMUNICATION_LIVETEST_DYNAMIC_CONNECTION_STRING);
 
-  return new ChatClient(url, new AzureCommunicationTokenCredential(userToken), {
-    httpClient: createTestHttpClient()
-  });
-}
-
-function createTestHttpClient(): HttpClient {
```

Comment:
you could still do the same using core v2 if you want.

Reply:
This was not used in the tests...